package.json

```diff
@@ -19,7 +19,7 @@
   "scripts": {
     "start": "pm2 start .pm2.json",
     "start:debug": "SILEX_DEBUG=true node dist/server/server.js",
-    "build": "npm run build:css && npm run build:html && npm run build:js:release ",
```

Comment:
if you remove this, the file `dist/client/js/admin.js` will not be generated, so Silex will not work at all